skiko/build.gradle.kts

```diff
@@ -45,6 +45,7 @@ allprojects {
 
 repositories {
     mavenCentral()
+    maven("https://packages.jetbrains.team/maven/p/ij/intellij-dependencies")
```

Comment:
It seems that it will be required for all Compose customers and it looks like a blocker here

cc @igordmn